### ISSUE TITLE
FAILED: ace-lm failed: Unknown option: --lm acestep.cpp 4922ed1 (2026-05-20)

### ISSUE THREAD

**Issue by zrukonf** [audiohacking/acestep-cpp-ui#34]
Hi, the generator isn't working, I don't understand what's wrong.
=== Job job_1781578126396_57jg0nj FAILED: ace-lm failed: Unknown option: --lm
acestep.cpp 4922ed1 (2026-05-20)

Usage: E:\ACE-Step-1.5\acestep-cpp-ui2\bin\ace-lm.exe --models <dir> --request <json> [options]

Required:
  --models <dir>         Directory of GGUF model files
  --request <json>       Input request JSON (carries lm_model)

Debug:
  --max-seq <N>          KV cache size (default: 8192)
  --no-fsm               Disable FSM constrained decoding
  --no-fa                Disable flash attention
  --no-batch-cfg         Split CFG into two separate forwards
  --clamp-fp16           Clamp hidden states to FP16 range
  --dump-logits <path>   Dump prefill logits (binary f32)
  --dump-tokens <path>   Dump prompt token IDs (CSV)
 ===
